packages/driver/src/cy/multi-domain/util.ts

```diff
@@ -0,0 +1,19 @@
+import _ from 'lodash'
+
+export const serializeRunnable = (runnable) => {
+  if (!runnable) return undefined
+
+  const fields = _.pick(runnable, ['id', 'type', 'title', 'parent', 'ctx', 'titlePath', '_timeout'])
```

Comment:
Should ctx be included here if it's overridden on line 8?

Reply:
I tried removing it and it screws up the inferred type of `fields`. I think it's better to leave it as-is instead of needing to write out the type explicitly so that TypeScript knows fields can have a `ctx` property.